web/src/screens/settings/Releases.tsx

```diff
@@ -66,7 +105,38 @@ function ReleaseSettings() {
               <h3 style={{ textAlign: "center" }} className="text-lg leading-6 font-medium text-gray-900 dark:text-white">
                 Danger Zone
               </h3>
-              <p style={{ textAlign: "center" }} className="mt-1 text-sm text-gray-900 dark:text-white">This will clear all release history in your database.</p>
+              <p style={{ textAlign: "center" }} className="mt-1 text-sm text-gray-900 dark:text-white">This will clear release history in your database.</p>
+            </div>
+            <div className="mt-6">
+              <label htmlFor="duration" className="items-center block text-sm font-medium text-gray-700 dark:text-white text-center">
```

Comment:
Another thing, I think it would be better if this text was left-aligned as opposed to centered.

Reply:
Thanks for all the feedback and suggestions as always 👍 

Can you show me a screenshot of what you mean perhaps? I'm not sure if you want the text all the way to the left. It'd be great to confirm!

Reply:
Was referring to the label "Delete releases older than:", so it's in line with the dropdown element text. It's a personal preference, however.

Looking at your screenshot again, might also be nice to change the subtitle on the panel page from "Release settings. Reset state." to something like "Manage release history" or something similar.

Reply:
Pushed the changes you suggested. New screenshot in the OP!

Reply:
Noice!